From 0d9c97476edc86a902f12ad2d596973308da54e0 Mon Sep 17 00:00:00 2001
From: Jayesh Choudhary <j-choudhary@ti.com>
Date: Fri, 7 Feb 2025 11:09:09 +0530
Subject: [PATCH 23/68] PENDING: ASoC: ti: j721e-evm: Reduce log level for
 EPROBE_DEFER

Drop the log level for deferral probe to avoid flooding in kernel logs.

Signed-off-by: Jayesh Choudhary <j-choudhary@ti.com>
---
 sound/soc/ti/j721e-evm.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/sound/soc/ti/j721e-evm.c b/sound/soc/ti/j721e-evm.c
index d9d1e021f5b2..f4f3b549c2a1 100644
--- a/sound/soc/ti/j721e-evm.c
+++ b/sound/soc/ti/j721e-evm.c
@@ -913,8 +913,9 @@ static int j721e_soc_probe(struct platform_device *pdev)
 	mutex_init(&priv->mutex);
 	ret = devm_snd_soc_register_card(&pdev->dev, card);
 	if (ret)
-		dev_err(&pdev->dev, "devm_snd_soc_register_card() failed: %d\n",
-			ret);
+		dev_err_probe(&pdev->dev, ret,
+			      "devm_snd_soc_register_card() failed: %d\n",
+			      ret);
 
 	return ret;
 }
-- 
2.47.2

